hera_sim/visibilities/vis_cpu.py

```diff
@@ -105,19 +119,21 @@ def _base_simulate(self, crd_eq, I):
         visfull = np.zeros_like(self.uvdata.data_array, dtype=self._complex_dtype)
 
         for i, freq in enumerate(self.freqs):
+
             vis = vis_cpu(
                 antpos=self.antpos,
-                freq=freq,
+                freq=self.freqs[i],
                 eq2tops=eq2tops,
                 crd_eq=crd_eq,
                 I_sky=I[i],
                 bm_cube=beam_lm[:, i],
                 real_dtype=self._real_dtype,
-                complex_dtype=self._complex_dtype
+                complex_dtype=self._complex_dtype,
+                c=self.c
```

Comment:
not sure why `c` is being passed... it's a constant

Reply:
Removed it. Was only there temporarily to check that `c` being calculated more or less accurately might have caused some of the phase differences between `vis_cpu` and `healvis` (which was not the case).